src/ImagePotentialBondGPU.h

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2018-2020, Michael P. Howard
+// Copyright (c) 2021-2025, Auburn University
+// Part of azplugins, released under the BSD 3-Clause License.
+
+#ifndef __IMAGE_POTENTIAL_BOND_GPU_H__
+#define __IMAGE_POTENTIAL_BOND_GPU_H__
+
+#ifdef ENABLE_HIP
+
+#include "ImagePotentialBondGPU.cuh"
+#include "hoomd/Autotuner.h"
+#include "hoomd/md/PotentialBondGPU.h"
+
+/*! \file ImagePotentialBondGPU.h
+    \brief Defines the template class for standard bond potentials on the GPU
+    \note This header cannot be compiled by nvcc
+*/
+
+#ifdef __HIPCC__
+#error This header cannot be compiled by nvcc
+#endif
+
+namespace hoomd
+    {
+namespace azplugins
+    {
+//! Template class for computing bond potentials on the GPU
+/*!
+    \tparam evaluator EvaluatorBond class used to evaluate V(r) and F(r)/r
+    \tparam Bonds Bond data type
+
+    \sa export_ImagePotentialBondGPU()
+*/
+template<class evaluator, class Bonds>
+class ImagePotentialBondGPU : public md::PotentialBondGPU<evaluator, Bonds>
+    {
+    public:
+    //! Inherit constructors from base class
+    using md::PotentialBondGPU<evaluator, Bonds>::PotentialBondGPU;
+
+    protected:
+    virtual void computeForces(uint64_t timestep);
```

Comment:
This needs to be overridden. You also need to reimplement the method requesting the image flags in MPI because that is not present in the class you're deriving from.
```suggestion
    void computeForces(uint64_t timestep) override;
```